ipynb/examples/trace_analysis/TraceAnalysis_IdleStates.ipynb

```diff
@@ -709,11 +818,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
+   "execution_count": 18,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'df' is not defined",
+     "output_type": "error",
```

Comment:
This reports failure here. It would be good to rerun it as it also has some missing plots at the end of the notebook.